spotify_player/src/client/mod.rs

```diff
@@ -1547,9 +1546,14 @@ impl Client {
     async fn retrieve_image(
         &self,
         url: &str,
-        path: &std::path::Path,
+        filename: &str,
         saved: bool,
     ) -> Result<Vec<u8>> {
+        let configs = config::get_config();
+        let hashedname = crate::utils::hash_filename(filename);
```

Comment:
you can append the hash at the end of the name instead. Also make it shorter, e.g use `6` characters instead of `16`? The main idea to keep the album's name and artist's name is to make it easy for users to find

Reply:
I used 6 digits of the album id as the hash now